packages/cli/CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+All notable changes to the [celo cli package](https://www.npmjs.com/package/@celo/celocli) will be documented in this file. 
+
+This package will follow the release process outlined [here](https://docs.celo.org/community/release-process).
```

Comment:
Note: The Release Process only has details for Smart Contracts and Blockchain Client. Should we also have separate release process guides in the documentation for each of the individual packages? Or a catch all if the other packages follow the same versioning?

Reply:
Oh wait, I see that's what one of the files is for already 👍